tests/console/vsftpd.pm

```diff
@@ -33,6 +33,7 @@ use utils 'zypper_call';
 sub run {
     my $self = shift;
     $self->select_serial_terminal;
+    my $arch = get_var('ARCH');
```

Comment:
if you import `Utils::Architectures 'is_s390x';` you can get rid of this variable